requirements/edx/edx-private.txt

```diff
@@ -8,7 +8,7 @@
 
 # For Harvard courses:
 -e git+https://github.com/gsehub/xblock-mentoring.git@4d1cce78dc232d5da6ffd73817b5c490e87a6eee#egg=xblock-mentoring
--e git+https://github.com/open-craft/problem-builder.git@90b2ecd8e948f34162e558b22f60e90287385f32#egg=xblock-problem-builder
+git+https://github.com/open-craft/problem-builder.git@v2.0.1#egg=xblock-problem-builder
```

Comment:
The instructions at the top of github.txt will explain the `==X.x.x` that you should have at the end of this URL.